src/java/install.sh

```diff
@@ -206,15 +206,21 @@ sdk_install() {
     local full_version_check=${5:-".*-[a-z]+"}
     local set_as_default=${6:-"true"}
     if [ "${requested_version}" = "none" ]; then return; fi
-     # Blank will install default version SDKMAN has
     if [ "${requested_version}" = "default" ]; then
          requested_version=""
     elif echo "${requested_version}" | grep -oE "${full_version_check}" > /dev/null 2>&1; then
         echo "${requested_version}"
     else
         local regex="${prefix}\\K[0-9]+\\.?[0-9]*\\.?[0-9]*${suffix}"
         local version_list=$(su ${USERNAME} -c ". \${SDKMAN_DIR}/bin/sdkman-init.sh && sdk list ${install_type} 2>&1 | grep -oP \"${regex}\" | tr -d ' ' | sort -rV")
-        if [ "${requested_version}" = "latest" ] || [ "${requested_version}" = "current" ] || [ "${requested_version}" = "lts" ]; then
+        if [ "${requested_version}" = "lts" ]; then
+            check_packages jq
+            all_lts_versions=$(curl -s https://api.adoptopenjdk.net/v3/info/available_releases)
```

Comment:
Won't this only work for the "Temurin" distribution?

I am concerned that a user might encounter failures if they select a different distribution that hasn't yet released the LTS version returned by this API.

Is there an easier way to find the LTS version for each JDK distribution?